### DISCUSSION TITLE
May I join you team

### DISCUSSION THREAD

**Discussion by NRanjan-17** [ROM-builders/temporary#9410] [General]
https://github.com/NRanjan-17/device_xiaomi_mido_pixel
https://github.com/NRanjan-17/proprietary_vendor_xiaomi
https://cirrus-ci.com/github/ROM-builders/temporary/pull/8603

May I join your team ?

**Comment by Apon77** [comment]
> https://github.com/NRanjan-17/device_xiaomi_mido_pixel https://github.com/NRanjan-17/proprietary_vendor_xiaomi https://cirrus-ci.com/github/ROM-builders/temporary/pull/8603
> 
> May I join your team ?

please check #learn in telegram group. 
thank you.
